src/recommendations/RecommendationsPage.jsx

```diff
@@ -46,8 +64,8 @@ const RecommendationsPage = () => {
 
   const handleSkipRecommendationPage = () => {
     window.history.replaceState(location.state, null, '');
-    if (registrationResponse) {
-      window.location.href = registrationResponse.redirectUrl;
+    if (redirectUrl) {
+      window.location.href = redirectUrl;
     } else {
       window.location.href = DASHBOARD_URL;
     }
```

Comment:
this would disturb the authn mfe registration flow and will not redirect users to redirect URL after the user registered from authn mfe.

Reply:
This will not disturb the authn MFE flow because we introduce a state and set `registrationResponse.redirectUrl` to `redirectUrl` as a initial state value and using it here in if condation.

Reply:
you are right.